.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name":"kendraio-app",
+    "forwardPorts": [4200],
+
+    "image": "mcr.microsoft.com/vscode/devcontainers/typescript-node:0-16"
```

Comment:
Why this specific image? @the-greenman

Reply:
I thought it was a good idea to pick a particular image that was optimised for our kind of work, rather than a general one. This is a step towards looking at our own docker config that would be optimised for our work.